src/types/index.ts

```diff
@@ -62,3 +62,8 @@ export interface Fraction {
   denominator: BN
   numerator: BN
 }
+
+export enum AnalyticsDimension {
+  NETWORK,
+  BROWSER_TYPE,
+}
```

Comment:
And why is this on root types?

I mean, looks like duplication. Either on theme or here?

Reply:
This is a generic enum with a few metrics that apps use. 
Each app define what's the dimension name in the context of the app. However, you can see how theres some generic components like `src/api/analytics/index.tsx` (initAnalytics). They don't know the name of the dimension, but they can tell if the app has specified a BROWSER_TYPE dimension or not.